src/components/atoms/Button/Button.jsx

```diff
@@ -5,6 +5,7 @@ import './Button.scss';
 export const Button = ({ text, variant, onClick }) => {
     return (
         <button
+            type='button'
```

Comment:
can you refactor this button, as a quick fix
Add button type as a prop (submit and button ~ 'button' should be default)